numpy/lib/tests/test_io.py

```diff
@@ -1823,6 +1828,123 @@ def test_auto_dtype_largeint(self):
         assert_equal(test['f1'], 17179869184)
         assert_equal(test['f2'], 1024)
 
+
+class TestPathUsage(TestCase):
+    @contextlib.contextmanager
```

Comment:
| then delete that file

Closing a file created by mkstemp does not delete it, that needs to be done separately.